### ISSUE TITLE
Can only build some sites

### ISSUE THREAD

**Issue by bogdanteleaga** [kfix/MacPin#29]
Tried doing a `make reinstall` as suggested in a different issue, but it gets stuck with the same error as the one in Hangouts below. The previous apps in the list build fine. Messenger doesn't build either, with some weird write error. I checked the permissions in the folder and they're fine. Slack also builds fine, which is the other one I was interested in. Do you have any idea for how to fix this?

I'm running 10.12.3 and got latest XCode and STP.

Oh, and any chances for a new release?

[Hangouts](https://gist.github.com/anonymous/089e41057901d4d79fac414b2e6de49d)
[Messenger](https://gist.github.com/anonymous/e21a993abf85a3447694606fe08dcd83)

**Comment by kfix** [comment]
try `make sites/Hangouts sites/Messenger`. No trailing slashes.

**Comment by bogdanteleaga** [comment]
Heh, seems to fix the Messenger one indeed. However, the Hangouts one is still broken, probably because it's a different error, related to the icon(?)

**Comment by bogdanteleaga** [comment]
I also seem to get these errors, even though the app is build and does start up. Not completely sure what the sealing is about.

```
Sealed Resources version=2 rules=13 files=26
# designated => cdhash H"d80c09530ae8e99d1258faa5dba7264efe9dc4c2" or cdhash H"77c07c827f2396e5f2ecbb06119d9a6be17bac0e"
<?xml version=1.0 encoding=UTF-8?>
<!DOCTYPE plist PUBLIC -//Apple//DTD PLIST 1.0//EN http://www.apple.com/DTDs/PropertyList-1.0.dtd>
<plist version=1.0>
<dict>
	<!--
		https://github.com/WebKit/webkit/bbuild/macosx-x86_64-apple-macosx10.11/apps/Messenger.app: a sealed resource is missing or invalid
make: [build/macosx-x86_64-apple-macosx10.11/apps/Messenger.app] Error 1 (ignored)
asctl container acl list -file build/macosx-x86_64-apple-macosx10.11/apps/Messenger.app
asctl: unable to read container for /private/tmp/boo/build/macosx-x86_64-apple-macosx10.11/apps/Messenger.app: /private/tmp/boo/build/macosx-x86_64-apple-macosx10.11/apps/Messenger.app is not Application Sandbox enabled
make: [build/macosx-x86_64-apple-macosx10.11/apps/Messenger.app] Error 71 (ignored)
Finished building build/macosx-x86_64-apple-macosx10.11/apps/Messenger.app
make: write error
```

**Comment by kfix** [comment]
`make: write error` was caused by `xctool`, believe it or not, and fixed by fedad862
`make reinstall` now runs thru all sites/ for me.
